### PR TITLE
Fix breadcrumbs

### PR DESCRIPTION
**Pull request by wheelsandcogs**
This PR needs the code in https://github.com/UKHomeOffice/asl-pages/pull/378 to be merged first and then bumped.

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ module.exports = settings => {
     if (isNaN(req.establishmentId)) {
       return next(new NotFoundError());
     }
+    res.locals.static.establishmentId = req.establishmentId; // needed for breadcrumbs
```
Comment (joefitter) [inline lib/app.js]: does this not get added to locals? I'm pretty sure we spread req.params to locals.static in the page router in service

Comment (wheelsandcogs) [reply lib/app.js]: I think it depends what route you hit - as the projectVersion routes don't seem to have it.

Comment (joefitter) [reply lib/app.js]: projectVersion edit routes in asl, or view in pages? or all of the above?

**Comment by joefitter** [comment]
closing as this has been resolved elsewhere

**Pull request closed without merging**